src/BroadcastSchedule.php

```
@@ -81,12 +81,23 @@ public function __construct()
             }
         }
 
+        // Explicit status: the payload is cached publicly, so an editor's
+        // request must not bake private shows into the schedule.
         $shows = Timber::get_posts([
             'post_type' => 'fm',
+            'post_status' => 'publish',
             'posts_per_page' => -1,
             'ignore_sticky_posts' => true,
         ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Timber\Timber' in method '__construct'. (undefined)

(StaticAccess)